_schema.graphql

```diff
@@ -8381,11 +8381,12 @@ type Show implements Node {
   # Does the show exist as a fair booth?
   is_fair_booth: Boolean
 
-  # Is it a stubbed show?
+  # Is it a show provided for historical reference?
   is_reference: Boolean
+  is_local_discovery: Boolean @deprecated(reason: "Prefer isStubShow")
```

Comment:
<!--
  1 failure:  Addition uses `sn...
  0 warning: 
  
  
  DangerID: danger-id-default;
  File: _schema.graphql;
  Line: 8386;
-->
- :no_entry_sign: Addition uses `snake_case` instead of `camelCase`: `is_local_discovery`

Reply:
Aha, will need to update the rule to look for a removal of the same addition in the same diff chunk.

Reply:
Neat inline commenting though!